### PR TITLE
Add french kissing

### PR DESCRIPTION
**Pull request by optimumtact**
If you wear the french beret you get the garlic breath trait added.

This trait makes your kisses become french and causes your breath to
smell of garlic

:cl: oranges
add: Added french kissing to the game
/:cl:

### CODE REVIEW

**Comment by Farquaar** [comment]
Rule 11 violation

File: code/game/objects/items/hand_items.dm [hunk 1]
```diff
@@ -378,3 +381,12 @@
...
 	var/obj/item/organ/heart/dont_go_breakin_my_heart = heartbreakee.getorganslot(ORGAN_SLOT_HEART)
 	dont_go_breakin_my_heart.applyOrganDamage(999)
+
+
+/obj/projectile/kiss/french
+	name = "french kiss (is that a hint of garlic?)"
+
+/obj/projectile/kiss/french/harmless_on_hit(mob/living/living_target)
+	. = ..()
+	living_target.visible_message("[living_target] has a funny look on their face", "Wow, that is a strong after taste of garlic!", vision_distance=COMBAT_MESSAGE_RANGE)
```
Comment (Mothblocks) [inline code/game/objects/items/hand_items.dm:391]:
```suggestion
	living_target.visible_message("[living_target] has a funny look on their face.", "Wow, that is a strong after taste of garlic!", vision_distance=COMBAT_MESSAGE_RANGE)
```

**Comment by Kylerace** [comment]
affects vampire balance

**Comment by JohnFulpWillard** [comment]
Please censor the word fr*nch, it is derogatory

**Comment by optimumtact** [comment]
> affects vampire balance

Killing vampires with kisses is how we do things around here

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 2]
```diff
@@ -417,6 +417,14 @@
 	metabolization_rate = 0.15 * REAGENTS_METABOLISM
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
+/datum/reagent/consumable/garlic/on_mob_add(mob/living/L, amount)
+	. = ..()
+	ADD_TRAIT(L, TRAIT_GARLIC_BREATH, src.type)
```
Comment (Fikou) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm:422]: dont use src.var sire!

Comment (optimumtact) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm:422]: UR NOT MY DAD

File: code/game/objects/items/hand_items.dm [hunk 3]
```diff
@@ -378,3 +381,13 @@
...
+
+
+/obj/projectile/kiss/french
+	name = "french kiss (is that a hint of garlic?)"
+
+/obj/projectile/kiss/french/harmless_on_hit(mob/living/living_target)
+	. = ..()
+	//Phwoar
+	living_target.reagents.add_reagent(/datum/reagent/consumable/garlic, 1)
+	living_target.visible_message("[living_target] has a funny look on their face.", "Wow, that is a strong after taste of garlic!", vision_distance=COMBAT_MESSAGE_RANGE)
```
Comment (Ryll-Ryll) [inline code/game/objects/items/hand_items.dm:394]:
```suggestion
	living_target.visible_message("[living_target] has a funny look on [living_target.p_their()] face.", "Wow, that is a strong after taste of garlic!", vision_distance=COMBAT_MESSAGE_RANGE)
```

**Comment by ATH1909** [comment]
the image of a mime defeating a vampire wizard by repeatedly blowing kisses at them is hilarious to me

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
image

**Comment by Cyberboss** [comment]
> Rule 11 violation

Your game rules can't save you here

**Comment by optimumtact** [comment]
I need to cap how much garlic the kisses can transfer because if you're a gourmand having garlic in your system provides some passive healing

**Comment by LemonInTheDark** [comment]
Let's gooooo

**Pull request merged**